homeassistant/components/overkiz/coordinator.py

```diff
@@ -135,6 +136,17 @@ async def on_device_unavailable_disabled(
         coordinator.devices[event.device_url].available = False
 
 
+@EVENT_HANDLERS.register(EventName.DEVICE_CREATED)
+@EVENT_HANDLERS.register(EventName.DEVICE_UPDATED)
+async def on_device_created_updated(
+    coordinator: OverkizDataUpdateCoordinator, event: Event
+) -> None:
+    """Handle device unavailable / disabled event."""
+    coordinator.hass.async_create_task(
+        coordinator.hass.config_entries.async_reload(coordinator._config_entry_id)
```

Comment:
Probably make the config entry id attribute non protected.

Reply:
1aed36fd8b4d5a91e312650e23923f009411fa3b